### PR TITLE
Learner profile fields

### PR DESCRIPTION
**Pull request by mbareta**

### CODE REVIEW

**Comment by mbareta** [comment]
TODO
1. ~~US states + Wash DC~~
2. ~~required/optional check~~

File: lms/templates/ccx/coach_dashboard.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
         <section class="instructor-dashboard-content-2" id="ccx-coach-dashboard-content" aria-labelledby="header-ccx-dashboard">
           <h2 class="hd hd-2" id="header-ccx-dashboard">${_("CCX Coach Dashboard")}</h2>
 
-          %if not ccx:
+          %if not ccx or True:
```
Comment (mbareta) [inline lms/templates/ccx/coach_dashboard.html:29]: Bonus: always show CCX create form.

Comment (vukoviciv) [reply lms/templates/ccx/coach_dashboard.html:29]: Is there a reason why we are not removing whole if condition?

**Comment by vukoviciv** [comment]
:+1:

**Pull request merged**